helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -51,8 +51,21 @@ starrocksOperator:
 starrocksCluster:
   # the name of starrockscluster cluster, if not set, the chart name will be used.
   name: ""
-  # the namespace of starrockscluster cluster, if not set, the release namespace will be used.
-  namespace: ""
+  # By default, starrockscluster will be deployed in the same namespace as the
+  # operator, and normally you can use helm --namespace to specify the namespace.
+  # This is used to create a different namespace for starrockscluster cluster with specified annotations.
+  namespace:
+    # if not set, the release namespace will be used.
+    # if set, the namespace will be created, and starrocks cluster and operator
+    # will be deployed in different namespaces.
+    name: ""
+    # annotations for starrockscluster namespace.
+    annotations: {}
```

Comment:
or rather than asking user to provide annotations here, can we have a `runOnOpenShift` option, if user sets it to `true`, chart automatically generate the annotation for the namespace.

Again, if the namespace already exists, will this change cause upgrade failure? or what exactly the behavior is?